Documentation/TypoScriptOverview/Prerequisites/Index.rst

```diff
@@ -18,15 +18,15 @@ familiar with the concepts of pages and content elements.
 
 A few more elements that you need to know before starting:
 
-- all content elements are stored in a database table called `tt_content`
+- all content elements are stored in a database table called :sql:`tt_content`
 
-- each content element has a database field called `CType` in which
+- each content element has a database field called :sql:`CType` in which
   the type of the content element is stored
 
-- the `tt_content` table also has a field called `pid` which refers
+- the :sql:`tt_content` table also has a field called :sql:`pid` which refers
```

Comment:
We currently do not have defined `sql` as inline syntax role. This will throw a warning during rendering.

I'll fix this after merge.